packages/odyssey/src/scss/base/_elements.scss

```diff
@@ -0,0 +1,83 @@
+// Note: This file contians empty, commented out style definitons. This means they were not overriden in anyway, but accounted for in documentation.
+
+$blockquote-border-left: 3px solid cv('gray', '200');
```

Comment:
Since these aren't being re-used and already use the standards foundations, we can skip the variables for now.

Reply:
For `blockquote` we have a semi-standard border-color: `$base-border-color`